packages/mobile/src/app/App.tsx

```diff
@@ -47,7 +47,9 @@ BigNumber.config({
 
 // Enables LayoutAnimation on Android
 if (Platform.OS === 'android') {
-  UIManager.setLayoutAnimationEnabledExperimental(true)
+  // tslint:disable-next-line:no-unused-expression
+  UIManager.setLayoutAnimationEnabledExperimental &&
```

Comment:
could avoid the tslint disabled by combining the android check

`if (Platform.OS === 'android' && UIManager.setLayoutAnimationEnabledExperimental) {